src/server/controllers/QuestionsController.ts

```diff
@@ -104,13 +104,14 @@ export class QuestionsController {
 
     const HOSTNAME = process.env.NEXT_PUBLIC_HOSTNAME;
 
+    if (!HOSTNAME) {
+      console.error("HOSTNAME environment variable is not set.");
+      return { error: "HOSTNAME environment variable is not set." };
```

Comment:
Hajde ovde da ubacimo samo jedan exception 

znaci umesto return, da stavimo 

throw new Error("HOSTNAME environment variable is not set."');